mantis-control-plane/mantis-control-plane-server/build.gradle

```diff
@@ -17,34 +17,34 @@
 apply plugin: 'application'
 
 ext {
-    akkaVersion = '2.5.23'
-    akkaHttpVersion = '10.1.8'
+    akkaVersion = '2.6.15'
```

Comment:
The newer ones are 2.6.18 and 10.2.9. Do you think we can advance to those ones instead (asking since I have those version upgrade in some of my code already).

Reply:
These are the versions that the flink RPC library brings in. I want the versions to match that.

Reply:
I see. Let me use these ones on my change too then.